src/sagemaker/workflow/README.rst

```diff
@@ -0,0 +1,162 @@
+
+    # trans_config specifies SageMaker batch transform configuration
+    trans_config = transform_config_from_estimator(estimator=estimator,
+                                                   instance_count=1,
+                                                   instance_type='ml.m4.xlarge',
+                                                   data=your_transform_data_s3_uri,
+                                                   content_type='text/csv')
+
+Now we can pass these configurations to related SageMaker operators and create the workflow:
+
+.. code:: python
+
+    import airflow
+    from airflow import DAG
+    from airflow.contrib.operators.sagemaker_training_operator import SageMakerTrainingOperator
+    from airflow.contrib.operators.sagemaker_transform_operator import SageMakerTransformOperator
+
+    default_args = {
+        'owner': 'airflow',
+        'start_date': airflow.utils.dates.days_ago(2),
+        'provide_context': True
+    }
+
+    dag = DAG('tensorflow_example', default_args=default_args,
+              schedule_interval='@once')
+
+    train_op = SageMakerTrainingOperator(
+        task_id='training',
+        config=train_config,
+        wait_for_completion=True,
+        dag=dag)
+
+    transform_op = SageMakerTransformOperator(
+        task_id='transform',
+        config=trans_config,
+        wait_for_completion=True,
+        dag=dag)
+
+    transform_op.set_upstream(train_op)
+
+Using Airflow Python Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Airflow PythonOperator <https://airflow.apache.org/howto/operator.html?#pythonoperator>`_
+is a built-in operator that can execute any Python callables. If you want to build the SageMaker workflow in a more
+flexible way,  you could write your python callables for SageMaker operations using SageMaker Python SDK. For example:
+
+.. code:: python
+
+    from sagemaker.tensorflow import TensorFlow
+
+    # callable for SageMaker training in TensorFlow
+    def train(data, **context):
+        estimator = TensorFlow(entry_point='tf_train.py',
+                               role='sagemaker-role',
+                               framework_version='1.11.0',
+                               training_steps=1000,
+                               evaluation_steps=100,
+                               train_instance_count=2,
+                               train_instance_type='ml.p2.xlarge')
+        estimator.fit(data)
+        return estimator.latest_training_job.job_name
+
+    # callable for SageMaker batch transform
+    def transform(data, **context):
+        training_job = context['ti'].xcom_pull(task_ids='training')
+        estimator = TensorFlow.attach(training_job)
+        transformer = estimator.transformer(instance_count=1, instance_type='ml.c4.xlarge')
+        transformer.transform(data, content_type='text/csv')
+
+Then you could build your workflow using PythonOperator with Python callables defined above:
```

Comment:
Then build your workflow by using the PythonOperator with the Python callables defined above:

Reply:
Updated.